### PR TITLE
use Intl.PluralRules instead of packaging CLDR

### LINKED ISSUE #172: Use Intl.PluralRules
[The Intl.PluralRules API](https://developers.google.com/web/updates/2017/10/intl-pluralrules) should allow this project to drop having to package CLDR in favor of offloading that to the browser or [polyfill](https://github.com/eemeli/IntlPluralRules).

### PR DESCRIPTION
**Pull request by longlho**
fixes #172 
also bumped major version since this is not backwards compat

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# 3.0.0
```
Comment (redonkulus) [inline CHANGELOG.md]: Looks like Releases has been used historically. Any reason not to use that?

Comment (longlho) [reply CHANGELOG.md]: oh snap, yeah I can definitely use that. But that'll be gone after the monorepo merge?

**Comment by longlho** [comment]
@redonkulus Intl.js polyfill already supports it (https://github.com/andyearnshaw/Intl.js/pull/262) like 2 years ago, but for some reason hasn't been released.

The other polyfill is https://github.com/eemeli/intl-pluralrules

We can def package a polyfill with ourselves. I don't mind either way

**Comment by redonkulus** [comment]
Could we just require the application to provide the polyfill instead of doing it here?

File: CHANGELOG.md (hunk 1 above)
Comment (okuryu) [reply CHANGELOG.md]: I prefer using GitHub Releases. I think we can make this file after migrated to monorepo If it's depends on monorepo.

**Comment by okuryu** [comment]
I hope Intl.js moves to FormatJS org to maintain parts.

File: CHANGELOG.md (hunk 1 above)
Comment (redonkulus) [reply CHANGELOG.md]: lerna also creates tags for each published package so we could continue to create releases for each package separately.

**Comment by redonkulus** [comment]
> I hope Intl.js moves to FormatJS org to maintain parts.

Intl.js was meant to be a stop gap solution until JS runtimes supported Intl natively. I believe most, if not all, modern browsers and Node support it today. We should start considering the general availability of these APIs instead of continuing to bundle or rely on a polyfill.

**Pull request merged**